Dockerfile

```diff
@@ -1,16 +1,18 @@
-FROM node:8.1.4
+FROM node:8.9.0-alpine
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-COPY package.json yarn.lock /usr/src/app/
-RUN yarn install
+COPY package.json yarn.lock tsconfig.json tsconfig.prod.json gulpfile.js /usr/src/app/
+RUN yarn install \
```

Comment:
Installing prod dependencies only and clearing cache reduces image size from around 1.1GB to 768MB. Swapping to alpine image reduces the size further to 161MB :)

Reply:
futher 161MB or to 161MB image size?

Reply:
To 161MB, that will be now the image size.

Reply:
By the way build sequence might be optimised a bit more, will do that on Monday.